### PR TITLE
Feat: Added missing templates: Jail, Bed and Colorfy

### PR DESCRIPTION
**Pull request by carefreeav09**
## Description
Added missing templates: Jail, Bed and Colofy.

## Motivation and Context
Some of the templates in the legacy build were missing. 

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/assets/TemplateFactory.ts [hunk 1]
```diff
@@ -499,5 +499,94 @@ export const TemplateFactory = {
...
+            {
+              source: new TemplateImage(image),
+              x: 0,
+              y: 0
+            }
+          ]
+        },
+        {
+          preprocess(canvas, ctx) {
+            ctx.fillStyle = color;
```
Comment (twlite) [inline src/assets/TemplateFactory.ts:517]:
```suggestion
            ctx.globalCompositeOperation = 'color';
            ctx.fillStyle = color;
```

`ctx.globalCompositeOperation = 'color';` is required for this to work properly

**Review by twlite — CHANGES_REQUESTED** [review]
Thank you for the contribution, I have left a review below.

**Review by twlite — APPROVED** [review]
LGTM!

**Comment by carefreeav09** [comment]
Updated as suggested!

**Pull request merged**